src/seedu/addressbook/data/tag/Tag.java

```diff
@@ -7,6 +7,8 @@
  * Guarantees: immutable; name is valid as declared in {@link #isValidTagName(String)}
  */
 public class Tag {
+	//implement tagging class
```

Comment:
The tagging class should be maintained in the Address  book as per the UML diagram given in the exercise.